doc/test/spell.en.pws

```diff
@@ -497,3 +497,18 @@ datetime
 formatter
 ndecode
 dothex
+cwd
+execve
+initrc
+INITRC
+loglevel
+overwite
```

Comment:
I think this is a spelling mistake: `overwite` -> `overwrite`.

Reply:
Ouch! Good catch!